File: 2nd year/Laboratories/MPP/project/data/src/main/java/com/alex/mpp/data/domain/Referee.java
package com.alex.mpp.data.domain;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.NonNull;
import lombok.RequiredArgsConstructor;

import java.io.Serializable;

@Data
@NoArgsConstructor
@RequiredArgsConstructor
public class Referee implements Serializable {
    private @NonNull Integer id;
    private @NonNull Integer gameId;
    private @NonNull String name;
    private @NonNull String username;
    private @NonNull String password;
}

File: 2nd year/Laboratories/MPP/project-cs/project/domain/Referee.cs
namespace project.domain
{
    public class Referee
    {
        public int Id { get; set; }
        public int GameId { get; set; }
        public string Name { get; set; }
        public string Username { get; set; }
        public string Password { get; set; }

        public override string ToString()
        {
            return
                $"{nameof(Id)}: {Id}, {nameof(GameId)}: {GameId}, {nameof(Name)}: {Name}, {nameof(Username)}: {Username}, {nameof(Password)}: {Password}";
        }
    }
}